activeadmin.gemspec

```diff
@@ -17,16 +17,16 @@ Gem::Specification.new do |s|
 
   s.extra_rdoc_files = %w[CHANGELOG.md CODE_OF_CONDUCT.md CONTRIBUTING.md README.md]
 
-  s.required_ruby_version = '>= 2.4'
+  s.required_ruby_version = '>= 2.5'
 
   s.add_dependency 'arbre', '~> 1.2', '>= 1.2.1'
-  s.add_dependency 'formtastic', '~> 3.1'
-  s.add_dependency 'formtastic_i18n', '~> 0.4'
-  s.add_dependency 'inherited_resources', '~> 1.7'
-  s.add_dependency 'jquery-rails', '~> 4.2'
-  s.add_dependency 'kaminari', '~> 1.0', '>= 1.0.1'
+  s.add_dependency 'formtastic', '>= 3.1'
+  s.add_dependency 'formtastic_i18n', '>= 0.4'
+  s.add_dependency 'inherited_resources', '>= 1.7'
+  s.add_dependency 'jquery-rails', '>= 1.0'
```

Comment:
Generally we prefer to use the `~>` operator. Specifying `~> x.y` means, higher than or equal to `x.y`, but also lower than `x+1.0`. This is good, because normally major versions include breaking changes, so we don't want to unrestrict the dependency in that way, because a major release of one of our dependencies could break `activeadmin`. This is why we normally use `~> x.y` rather than just `>= x.y`.

Reply:
I also tried that, and bundler would install the older versions, breaking the rest of the installation. Causing version trouble down the line.